index.html

```diff
@@ -319,5 +320,21 @@ <h3><i class="lni lni-envelope"></i><strong>Leave</strong> a Message.</h2>
       </section>
     </main>
     </main>
+
+    <footer>
+
+      <img src="assets/img/logo-2.svg">
+      <div class="social_media_icons">
```

Comment:
change class name to `social`
we're using `-` to separate words, e.g. `social-media` (but 'social' here is enough) and use BEM for naming